README.md

```diff
@@ -88,7 +88,7 @@ Please do, we like reported issues and pull requests.
 ## License
 MIT License
 
-Copyright (c) 2020 SIL International
+Copyright (c) 2022 SIL International
```

Comment:
(I'm no lawyer but you can probably keep this as 2020, the year you originally authored the work. At least that's what I do, it can be handy for seeing when something was written.)